.gitlab/pipeline/build.yml

```
@@ -329,6 +329,14 @@ build-linux-substrate:
     # - printf '\n# building node-template\n\n'
     # - ./scripts/ci/node-template-release.sh ./artifacts/substrate/substrate-node-template.tar.gz
 
+build-minimal-runtime-polkavm:
+  stage: build
+  extends:
+    - .docker-env
+    - .common-refs
+  script:
+    - SUBSTRATE_RUNTIME_TARGET=riscv cargo check -p minimal-node
```

```suggestion
    - SUBSTRATE_RUNTIME_TARGET=riscv cargo check -p minimal-runtime
```
Not sure why you want to check the entire node :D